Coding/server/lib/FileManager/FileManager.h

```diff
@@ -42,7 +42,118 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef __FILEMANAGER_H__
 #define __FILEMANAGER_H__
 
+#include <Arduino.h>
+#include <Log.h>
+#include <FS.h>
+#include <SPIFFS.h>
+
+ /** Class used for simply reading and writing binary files */
 class FileManager
 {
+public:
+    /** Enum for specifying the different access modes */
```

Comment:
No longer necessary if you use constants as described above.

Reply:
Fixed